content/http-handler-abi.md

```diff
@@ -767,6 +779,8 @@ method to "POST".
 ;; get_uri writes the URI to memory if it isn't larger than `buf_limit`, e.g.
 ;; "/v1.0/hi?name=panda". The result is its length in bytes.  
 ;;
+;; Note: If no URI have been set by set_uri, get_uri will return `/`.
```

Comment:
This isn't necessarily about a call to set_uri as this is a server handler. It is about whatever the underlying URI is. Ex if the incoming request (which wasn't using wasm to generate) had no URI and the specific host library represents "" instead of "/"

```suggestion
;; Note: The host should return "/" instead of empty for a request with no URL.
```

Reply:
So what I understand is that it is programmer responsibility to check if the URL is actually `""` and return `/` instead?

```suggestion
;; Note: The host should return "/" instead of empty for a request with no URL or empty URL.
```

Reply:
No the meaning is that the host should not return "" (the host side implementation), rather "/". Some hosts will never return "", so rather than have incompatible behavior based on host library, decide on "/" instead of sometimes "/" and sometimes "" depending on library.

The client will never see empty in other words.